packages/main/src/MenuItemTemplate.tsx

```diff
@@ -30,9 +30,22 @@ function listItemContent(this: MenuItem) {
 		{this.text && <div class="ui5-menu-item-text">{this.text}</div>}
 
 		{rightContent.call(this)}
+		{checkmarkContent.call(this)}
 	</>);
 }
 
+function checkmarkContent(this: MenuItem) {
+	return !this._markChecked ? "" : (
+		<div class="ui5-menu-item-checked">
+			<Icon
+				part="checked"
+				name="accept"
```

Comment:
Best practice - this icon needs to be imported by name and used here, f.e. same as `navBackIcon`